hicommonwealth/commonwealth
# Added valentine growl

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6636

## Description of Changes
- Added Valentine's growl
<img width="322" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/0bfb463d-b5b7-4b95-b0b6-999fd811aaea">


## "How We Fixed It"

N/A

## Test Plan
- Visit any page
- Verify you see the valentine's growl
- Click on the 'Add to Calender' button, verify it opens OS calender and adds event entry.
- Try to close it from the x button, verify it closes and shows again on page reload
- Click on the "Don't show this again" checkbox, and close it again from the x button, verify it closes and doesn't show again on page reload.

## Deployment Plan
N/A

## Other Considerations
N/A
